extensions/amp-list/0.1/amp-list.js

```diff
@@ -632,7 +633,7 @@ export class AmpList extends AMP.BaseElement {
       if (this.bind_ && this.bind_.signals().get('FIRST_MUTATE')) {
         return updateWith(this.bind_);
       } else {
-        return Promise.resolve(elements);
+        return Promise.resolve(/** @type !Array<!Element> */ (elements));
```

Comment:
Minor: Prefer to avoid typecasts where possible. Try using a new local var, e.g.

```js
updateBindings_(elementOrElements) {
  const elements = isArray(elementOrElements) ? elementOrElements : [elementOrElements];
  ...
}
```

Reply:
done.